### DISCUSSION TITLE
Infinite reloading loop after logging into the frontend

### DISCUSSION THREAD

**Discussion by Elektra58** [keycloak/keycloak#35713] [Q&A]
I have a full-stack application with **FastAPI/Python** in the backend and **React/TypeScript/Tailwind/Vite** in the frontend. After two full days I am still stuck in an infinite reloading loop. Does anybody have a minimal working app with React, FastAPI and the appropriate Keycloak json files?

**Comment by embesozzi** [comment]
Hi @Elektra58, in order to help, please describe the OAuth 2.0 / OIDC flows used, error logs, libraries used, etc.
If you are in the loop in the context of identity federation with OAuth 2.0 / OIDC, the error in the browser networking will give a hint of what's going on. It's common to encounter this kind of error when the app/library triggers a redirection to the IdP again due to a misconfiguration error.
